deploy/helm/templates/NOTES.txt

```diff
@@ -96,3 +96,8 @@ WARNING: Configuration `prometheus-node-exporter` was deprecated. Please remove
 WARNING: Value `otel.metrics.swi_endpoint_check` was deprecated and will be removed in a future release. Please use `otel.swi_endpoint_check.enabled` instead.
 {{- println -}}
 {{- end -}}
+
+{{- if .Values.autoupdate.enabled -}}
```

Comment:
Please add a test for this in https://github.com/solarwinds/swi-k8s-opentelemetry-collector/blob/master/deploy/helm/tests/notes_test.yaml